README.md

```diff
@@ -1,24 +1,142 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 Blog App <a name="about-project"></a>
 
-* Configuration
+> The Blog app is a classic example of a blog website. It's a functional website that shows the list of posts and empower readers to interact with them by adding comments and liking post.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">Ruby</a></li>
+  </ul>
+    <ul>
+    <li><a href="https://ruby-doc.org/core-3.1.2/">Rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Shows the list of posts**
+- **Shows the list of comments**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Install ruby on your computer or run in with docker
+- Run `ruby main`
+
+### Setup
+
+Clone this repository by using the command line :
+
+- `git clone https://github.com/studiosnabeel/blog-app.git`
+
+### Install
+
+Install this project with:
+
+`npm gemfile`
+
+### Usage
+
+To run the project, execute the following command:
+
+`ruby main`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Nabeel Akbar**
+
+- GitHub: [@githubhandle](https://github.com/studiosnabeel)
+- Twitter: [@twitterhandle](https://twitter.com/StudiosNabeel)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/studiosnabeel/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add More methods**
```

Comment:
- [ ] [optional] Kindly be specific on future methods (features) you desire to implement in the future.